### DISCUSSION TITLE
Item preview / popup

### DISCUSSION THREAD

**Discussion by timlind** [community/community#6500] [Projects and Issues] [answered]
A missing feature in the new Projects Beta is the issue / PR preview / popup that opens from the right on the old Projects.

This minimises context switching, and is especially useful when screen-sharing.

**Comment by ctwhome** [comment]
Please prioritize this feature to not change contexts everytime you want to peek to the details of the issue from the board or list

**Reply by mrcarter7** [reply]
This is 100% needed - I'm regretting starting my new project as a beta one without this feature

**Comment by sangbahri** [comment]
+1 to have this feature added to the new project beta.

**Comment by r4fx** [comment]
duplicate of https://github.com/github/feedback/discussions/5381, please vote there 🙂

**Comment by 2percentsilk** [comment, marked as answer]
This has been addressed with the release of the [side panel](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/) in Projects!
